dependencies/bionic/gitlab-sidekiq-fetcher/control

```diff
@@ -0,0 +1,16 @@
+Source: ruby-gitlab-sidekiq-fetcher
+Section: ruby
+Priority: optional
+Maintainer: Adam Ruzicka <aruzicka@redhat.com>
+Uploaders: Adam Ruzicka <aruzicka@redhat.com>
+DM-Upload-Allowed: yes
+Build-Depends: debhelper, gem2deb
+Standards-Version: 3.9.3
+XS-Ruby-Versions: all
+
+Package: ruby-dynflow
```

Comment:
This looks odd. Are you sure this package is generated from the source?

Reply:
Odd indeed. It wasn't generated per-se, I created the package by copying dynflow over and changing the contents. Looks like I missed this when editing it